unit/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -119,6 +119,18 @@ TEST_CASE("\"symbol_exprt\" to smt term conversion", "[core][smt2_incremental]")
     smt_identifier_termt("foo", smt_bool_sortt{}));
 }
 
+TEST_CASE("\"nondet_symbol_exprt\" to smt term conversion", "[core][smt2_incremental]")
+{
+    auto test = expr_to_smt_conversion_test_environmentt::make(test_archt::i386);
+    // Checking that the nondet_symbol is translated correctly to a term lookup
```

Comment:
:pick: This comment seems to say the same as what the code immediately below it already explicitly says. Do you think it is still worthwhile despite this duplication?

Reply:
Yes, the code documents itself in this case.
Removed